runtime/onert/core/include/util/logging.h

```diff
@@ -64,4 +64,10 @@ static Context &ctx = Context::get();
   if (::onert::util::logging::ctx.enabled()) \
   std::cout << "[" << __func__ << "] "
 
+#define WHENLOGENABLED(METHOD)               \
```

Comment:
@seanshpark What about `WHEN_LOG_ENABLED`? Or `onert` may have some convention for macro name though I don't remember.